### ISSUE TITLE
Missing return

### ISSUE THREAD

**Issue by gravic07** [udacity/ud330#9]
When checking for an error in the access token info, the response is not returned after line 73.

**Comment by SteveWooding** [comment]
I agree too. The comment says to abort and all the other times we make a response we return. I've added it to my copy. The video misses it too. The video also missed another `return response` line that the GitHub code caught. This is probably another case. Needs fixing I think.
